### PR TITLE
[CHI-251] Add support for light class

### PR DESCRIPTION
**Pull request by davidojedacl**
@mattnickles @jllr

### CODE REVIEW

File: src/chi/components/buttons/buttons.scss [hunk 1]
```diff
@@ -539,10 +546,240 @@ $sizes: (
...
+
+          &.-icon {
+            & .a-icon {
+              color: set-color(grey, 50);
+            }
+
+            &:hover,
+            &.-hover {
+              background-color: transparent;
+              border: $border solid transparent;
```
Comment (mattnickles) [inline src/chi/components/buttons/buttons.scss]: There's a bug with the focus state. Border is getting overridden on :hover:focus with this transparent border color value.

**Comment by jllr** [comment]
@davidojedacl Chi header is still using -inverse instead of -light

**Comment by mattnickles** [comment]
@jllr @davidojedacl _"Chi header is still using -inverse instead of -light"_ Let's only focus on buttons and badges for this PR. No need to change the -inverse class on o-header

File: src/website/views/components/badge.pug [hunk 2]
```diff
@@ -32,8 +32,8 @@ h4 Base
       .-p--2.-bg--black.-text--center
         .a-badge(class=`-secondary -m--1`)
           span Secondary
-        .a-badge(class=`-inverse -m--1`)
-          span Inverse
+        .a-badge(class=`-light`)
```
Comment (mattnickles) [inline src/website/views/components/badge.pug]: Margin was removed. Need to add back

File: src/website/views/components/badge.pug [hunk 3]
```diff
@@ -86,8 +86,8 @@ h4 Outline
       .-p--2.-bg--black.-text--center
         .a-badge(class=`-secondary -outline -m--1`)
           span Secondary
-        .a-badge(class=`-inverse -outline -m--1`)
-          span Inverse
+        .a-badge(class=`-light -outline`)
```
Comment (mattnickles) [inline src/website/views/components/badge.pug]: Margin was removed. Need to add back

File: src/website/views/components/badge.pug [hunk 4]
```diff
@@ -134,8 +134,8 @@ h4 Flat
       .-p--2.-bg--black.-text--center
         .a-badge(class=`-secondary -flat -m--1`)
           span Secondary
-        .a-badge(class=`-inverse -flat -m--1`)
-          span Inverse
+        .a-badge(class=`-light -flat`)
```
Comment (mattnickles) [inline src/website/views/components/badge.pug]: Margin was removed. Need to add back

File: src/website/views/components/badge.pug (hunk 2 above)
Comment (davidojedacl) [reply src/website/views/components/badge.pug]: done

File: src/website/views/components/badge.pug (hunk 3 above)
Comment (davidojedacl) [reply src/website/views/components/badge.pug]: done

**Pull request merged**